purchases/src/main/kotlin/com/revenuecat/purchases/PurchaserInfo.kt

```diff
@@ -32,7 +33,8 @@ class PurchaserInfo internal constructor(
     internal val jsonObject: JSONObject,
     internal val schemaVersion: Int,
     val firstSeen: Date,
-    val originalAppUserId: String
+    val originalAppUserId: String,
+    val managementURL: String?
```

Comment:
maybe it's a good idea to make this a `URL?` instead of a `String?` - it'd provide some guarantees for the content and be easier to use for devs

Reply:
yes! I will change it

Reply:
I changed it with Uri, what do you think?